containers/session-containers/skaha-desktop/Dockerfile.centos.xfce.vnc

```diff
@@ -115,16 +103,18 @@ RUN useradd -m -g guest -u 30001 guest && \
     ln -s /home/guest/.ssl /headless/.ssl
 
 # Add scripts that launch sofware containers
-ADD tmp/*.sh /usr/local/bin/
 ADD software-scripts/*.sh /usr/local/bin/
+RUN rm /usr/local/bin/dragons.sh
+RUN rm /usr/local/bin/nifty.sh
```

Comment:
And remove this section?  (lines 105-108)

Reply:
These .sh files are different from the generic ones. They are needed until we don't need these special .sh scripts.

Reply:
Will need to modify the those two applications so that they can take the generic .sh arguments to start up.

Also these two apps are currently not in the list of desktop-app. To test the change, it's better to include them in the list.

Reply:
Removed those two lines and the associated .sh files.